src-ts/FeeRouter/ChildToParentMessageReedeemer.ts

```diff
@@ -0,0 +1,101 @@
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { Wallet } from "ethers";
+import {
+  ChildToParentRewardRouter__factory,
+  ChildToParentRewardRouter,
+} from "../../typechain-types";
+import { L2ToL1TxEvent } from "@arbitrum/sdk/dist/lib/abi/ArbSys";
+import { EventArgs } from "@arbitrum/sdk/dist/lib/dataEntities/event";
+
+import {
+  L2TransactionReceipt,
+  L2ToL1Message,
+  L2ToL1MessageStatus,
+} from "@arbitrum/sdk";
+const wait = async (ms: number) => new Promise((res) => setTimeout(res, ms));
+
+export default class ChildToParentMessageReedeemer {
```

Comment:
Why call this one a redeemer and the other one a router?

Reply:
It only redeems them, doesn't initiate them? (idk)